docs/source/tutorials/extending_mantid_with_python/advanced_fit_behaviours/01_fit_stability.rst

```diff
@@ -0,0 +1,97 @@
+.. _01_fit_stability:
+
+=============
+Fit Stability
+=============
+
+In most cases, a fit will be performed over the parameters exactly as they
+are declared. In some cases, however, the fit can be unstable in one or more
+of the parameters. As an example consider a Gaussian:
+
+.. math::
+
+    Ae^{-\frac{(x-c)^2}{2σ^2}}
```

Comment:
Can you make this a little larger? The exponent could be a little small on some screens